ntfsutils/fs.py

```diff
@@ -85,12 +85,8 @@ class BY_HANDLE_FILE_INFORMATION(ctypes.Structure):
 CloseHandle.argtypes = [HANDLE]
 CloseHandle.restype = BOOL
 
-def getfileinfo(path):
-    """
-    Return information for the file at the given path. This is going to be a
-    struct of type BY_HANDLE_FILE_INFORMATION.
-    """
-    hfile = CreateFile(path, GENERIC_READ, FILE_SHARE_READ, None, OPEN_EXISTING, 0, None)
+def _getfileinfo(path, flags=0):
```

Comment:
small nit: I don't think a default argument is very useful here -- could you have each caller pass in the flags explicitly? That would be a little clearer.

Other than that this looks great. Thank you so much!

Reply:
Sure, will update in a second. Thank you!